.ci/test-project/uberenv_configs/install.json

```diff
@@ -5,6 +5,7 @@
 "spack_build_mode" : "install",
 "spack_configs_path": "spack_configs",
 "spack_packages_path": "packages",
+"spack_not_buildable_packages": "cmake pkg-config zlib",
```

Comment:
I don't like this name but can't come up with a better one right now. Let me ponder this.

Reply:
spack_packages_to_find ?

Reply:
`spack_externals` ?